NEWS.md

```diff
@@ -1,7 +1,33 @@
 # metalite.ae 0.1.2
 
-- Updated `format_ae_specific` to avoid warning when no comparison variables are
-requested.
+## New features
+- `tlf_ae_summary()` and `tlf_ae_specific` now have a new argument `title` for a user-customized title.
+- Add new functions: `rate_compare_sum()`, `extend_ae_specific_subgroup()`, `prepare_ae_specific_subgroup()`, `format_ae_specific_subgroup()`, `tlf_ae_specific_subgroup()`, `extend_ae_summary_eaer()`, 
+`format_ae_exp_adj()` and `tlf_ae_exp_adj()`.
+- Add `data/metalite_ae_adex.Rda`.
```

Comment:
For `.rda`, the convention is to use all lower case `.rda`. If you data file name actually ends with `.Rda`, you need to rename it.